fastlane/lanes/release.rb

```diff
@@ -88,6 +88,8 @@
     update_frozen_strings_for_translation
     download_translations()
     android_bump_version_beta()
+    return unless UI.confirm('Ready for CI build')
+
```

Comment:
I added that quickly so that it would pause and give me the opportunity to manually fix the `versionCode` bumps (in c8d07241ac1917e2097df5fc53b6af119aa472ef) to account for the `19.7.1` hotfix consuming the code `1211`.

I might actually keep it for the future, even if it's mostly useful for hotfixes, it doesn't hurt to have it.